crates/astria-composer/src/searcher/collector.rs

```diff
@@ -36,7 +36,7 @@ type StdError = dyn std::error::Error;
 /// passing them downstream for the searcher to process. Thus, a searcher can have multiple
 /// collectors running at the same time funneling data from multiple rollup nodes.
 #[derive(Debug)]
-pub(super) struct Collector {
+pub(super) struct GethCollector {
```

Comment:
can you remove these changes that were from your other PR?

Reply:
yes i will do that. I thought i would work these changes in order i.e in this order https://github.com/astriaorg/astria/issues/795 . Merge each PR to main. 

What do you think?